appveyor.yml

```diff
@@ -7,8 +7,6 @@ init:
 build_script:
   - ps: Invoke-RestMethod -Uri 'https://dot.net/v1/dotnet-install.ps1' -OutFile '.\dotnet-install.ps1'
```

Comment:
Do we need `dotnet-install.ps1` after the changes?

Reply:
Yes, as .NET 7 is in global.json we should install .NET 6.0, so remove 3.1 and replace 5.0 with 6.0.